src/screens/authentication/UsernameRequest/index.js

```diff
@@ -100,10 +119,42 @@ const UsernameRequest = (props) => {
     setError(null)
   }
 
+  const onSuggestionSelected = (suggestion) => {
+    setTextInputValue(suggestion)
+    setError(false)
+  }
+
+  useEffect(() => {
+    AnalyticsService.setCurrentScreenName('Username Request')
+  }, [])
+
+  useEffect(() => {
+    const createSuggestions = async () => {
+      let posibleUsernames = []
+      let i = 0
+
+      while (i < 9) {
+        posibleUsernames[i] = (textInputValue + '_' + Math.floor((Math.random() * 100) + 1))
```

Comment:
`9` and `100` are magic numbers, which means they serve a purpose but I have to read the code to know what they mean by context. Were they handpicked? They represent a threshold?

I understand the code, but it probably would improve it's readabilty to define

`const UPPER_USERNAME_LIMIT = 100` 
`const NUMBER_OF_POSSIBLE_USERNAMES = 9`

or something like that.

Reply:
Yeah the in function in firebase queries only let's you pass 10 parametres so the 9 was from there, i'll make the changes